scripts/python/policia_nacional/README

```diff
@@ -0,0 +1,52 @@
+		 + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +
+		+Conversion de PDF a CSV con tabula-py para datos de la policia nacional en la plataforma Control Ciudadano+
+		 + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + 
+
+#REQUISITOS
+
+- Necesita la instalacion de tabula-py 
```

Comment:
The requirements of python scripts are normally specified in a file called `requirements.txt`, see https://github.com/InstIDEA/controlciudadano/blob/master/scripts/python/parser/requirements.txt for example